requirements-app.txt

```diff
@@ -2,7 +2,8 @@
 # with package version changes made in requirements-app.txt
 
 aws-wsgi==0.2.7
-boto==2.49.0
+boto3==1.19.4
+botocore==1.22.4
```

Comment:
Can you run ` make freeze-requirements` and make sure everything is still good?

Reply:
Yes! that is my current goal, with some local testing with api + admin + celery.